jf_agent/git/github.py

```diff
@@ -39,18 +39,12 @@ def load_and_dump(
     # Logic to help with debugging the scopes that clients have with their API key
     try:
         scopes = git_conn.get_scopes_of_api_token()
-        agent_logging.log_and_print(
-            logger,
-            logging.INFO,
+        logger.debug(
             'Attempting to ingest github data with the following '
             f'scopes for {config.git_instance_slug}: {scopes}',
         )
     except Exception as e:
-        agent_logging.log_and_print(
-            logger,
-            logging.INFO,
-            f'Problem finding scopes for your API key. This should not affect your github agent processing. Error: {e}',
-        )
+        logger.debug(f'Problem finding scopes for your API key. Error: {e}', exc_info=True)
```

Comment:
Q: Similarly, why `debug` over `info` or even `warning`?

Reply:
Same reasoning as above. This is a sort of hacky thing I threw in to check scopes of peoples API tokens, to see if they were eligible for graphql cutovers. GQL cutovers are still happening, so I'd like to leave it in. This information is only really valuable for us, imo, so I'm setting it as DEBUG which will hide it from STDOUT